tests/unit/helpers.py

```diff
@@ -33,3 +33,11 @@ def cli_arg(plan, cli_opt):
     _patch=tautology,
     is_ready=tautology,
 )
+
+
+class FakeProcessVersionCheck:
```

Comment:
Should we mock `_alertmanager_version` instead?

Reply:
I sort of feed like it's best practice to mock the lowest level call. That way you exercise more of your code.